R/query.R

```diff
@@ -72,7 +72,7 @@ do_query <- function(query, server.url = NULL, timeout = NULL, print.json = FALS
                 if(httr::status_code(x) == 200) {
                     x <- httr::content(x, simplifyVector = TRUE)
                     if(x$status %in% c("fail-query", "fail-malformed-query", "fail-error", "fail-memory"))
-                        stop(sprintf("Query failed with status:%s\nerror message:%s", query.status, x$"error-message"))
+                        stop(sprintf("Query failed with status:%s", x$status))
```

Comment:
Can you elaborate on this? What's going on?

Reply:
Not sure what the old code was trying to do exactly, but `query.status` is never defined and was returning an undefined variable error instead of the status message like it was supposed to do.

Reply:
This has now been fixed in a different PR.